install/install.sh

```diff
@@ -87,7 +87,7 @@ verbose "Removing ${tarball}"
 rm $tarball
 
 verbose "Linking ${latest_directory} to ${target_directory}"
-ln -sf $target_directory $latest_directory
+ln -snf $target_directory $latest_directory
```

Comment:
Perhaps a comment noting the importance of `n` and `f`. (Or even a link to this very PR since it has a nice description ;)